components/BottomAppBar/src/MDCBottomAppBarView.m

```diff
@@ -323,10 +323,11 @@ - (void)setFloatingButtonElevation:(MDCBottomAppBarFloatingButtonElevation)float
   }
   if (animated) {
     [_floatingButton setElevation:1 forState:UIControlStateNormal];
+    __weak typeof(self) weakSelf = self;
     dispatch_after(dispatch_time(DISPATCH_TIME_NOW, kMDCButtonAnimationDuration * NSEC_PER_MSEC),
                    dispatch_get_main_queue(), ^{
-                     [self insertSubview:self.floatingButton atIndex:subViewIndex];
-                     [self.floatingButton setElevation:elevation forState:UIControlStateNormal];
+                     [weakSelf insertSubview:weakSelf.floatingButton atIndex:subViewIndex];
```

Comment:
I think you would want to strongify here.  Calling `-insertSubview:atIndex:` cannot accept a `nil` view.  Even though I believe the right hand side is evaluated first (leaving the receiver `nil`), it still feels like a risk.